apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/Trace.cs

```diff
@@ -55,15 +57,15 @@ namespace Google.Devtools.AspNet
     /// 
     /// Docs: https://cloud.google.com/trace/docs/
     /// </remarks>
-    public sealed class CloudTrace
+    public sealed class Trace
```

Comment:
This isn't I switched it back,  I think I should find a better name then CloudTrace eventually.